### PR TITLE
Use separate Dockerfile for Python 3, built from python3.6 base image

### PR DESCRIPTION
**Pull request by diox**
Can't use Python 3.7 yet because of issues with celery.

Fixes https://github.com/mozilla/addons-server/issues/10472

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -36,12 +36,11 @@ services:
   web3:
     extends:
       service: web
+    image: addons/addons-server-py3
```
Comment (diox) [inline docker-compose.yml]: We need to create this on docker hub once this PR is merged, have it point to this repo using `Dockerfile.python3`

File: Dockerfile.python3 [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+# This prevents dupes but only in memory for the current session.
+ENV HISTCONTROL erasedups
+
+ENV CLEANCSS_BIN /deps/node_modules/.bin/cleancss
+ENV LESS_BIN /deps/node_modules/.bin/lessc
+ENV UGLIFY_BIN /deps/node_modules/.bin/uglifyjs
+ENV ADDONS_LINTER_BIN /deps/node_modules/.bin/addons-linter
+ENV ADDONS_VALIDATOR_BIN /usr/local/bin/addon-validator
+
+ENV PYTHON_VERSION_MAJOR 3
```
Comment (eviljeff) [inline Dockerfile.python3]: should this be further up?  There's a `make` command on line 72

File: docker-compose.yml (hunk 1 above)
Comment (diox) [reply docker-compose.yml]: Alternatively, to test this PR locally, fetch the code and:
`docker-compose stop web3`
`docker build -f Dockerfile.python3 -t addons/addons-server-py3 .`
`docker-compose up -d web3` 

The `web3` container should now be the new container with Python 3.6.

File: Dockerfile.python3 (hunk 2 above)
Comment (diox) [reply Dockerfile.python3]: 54f131d7f7ad6952338ffb44af24b6c4c5e9a577

**Pull request merged**